.github/workflows/push-tests.yml

```diff
@@ -22,6 +22,12 @@ jobs:
   unit-test:
     runs-on: ubuntu-latest
     steps:
+    - name: Configure AWS Credentials
```

Comment:
This is great! This was one of the blockers to running functional tests during push tests. I'll take a bit of time to look into it again after this gets merged

Reply:
You will need to update this policy with the correct permissions https://github.com/chanzuckerberg/dcp-prototype/blob/d0a4c0fd6a4ff45ba86f036812ebe111714f601c/infra/modules/backend/cicd/github_actions_unittests.tf#L14